locale/en-us/dialog/en/shops_amount.dialog

```diff
@@ -0,0 +1 @@
+I found {{n}} {{store_name}}.
```

Comment:
Consider "I found {{n}} locations for {{store_name}}." to clarify these are locations for the same store and not different stores that sound similar.